app/controllers/resource_languages_controller.rb

```diff
@@ -0,0 +1,26 @@
+class ResourceLanguagesController < SecureController
+  before_action :get_resource, :get_language, :build_resource_language
+
+  def show
+    render json: @resource_language, include: params[:include], status: :ok
+  end
```

Comment:
can we make `show` public, but keep `update` private?

Reply:
Good catch, thanks

Reply:
Sure